src/Microsoft.ServiceFabric.Client.Http/LocalImageStoreClient.cs

```diff
@@ -0,0 +1,274 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.ServiceFabric.Client.Http
+{
+    using System;
+    using System.Collections.Concurrent;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.IO.Compression;
+    using System.Linq;
+    using System.Net.Http;
+    using System.Net.Http.Headers;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using System.Xml;
+    using System.Xml.Linq;
+    using System.Xml.XPath;
+    using Microsoft.ServiceFabric.Client;
+    using Microsoft.ServiceFabric.Client.Exceptions;
+    using Microsoft.ServiceFabric.Client.Http.Serialization;
+    using Microsoft.ServiceFabric.Common;
+    using Microsoft.ServiceFabric.Common.Utilities;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Class containing methods for performing ImageStoreClient operataions.
+    /// </summary>
+    internal partial class LocalImageStoreClient : IImageStoreClient
```

Comment:
IImageStoreClient has methods for REST calls available for ImageStoreService, some methods dont apply to LocalImageStore. I believe the correct way is to perform these operations of Copy, Delete differently in the Copy,Delete methods based on the image store being used,

Reply:
I'm not sure i understand what you mean, they are different in the different implementations?